### PR TITLE
Update version string to 22.xx.BETA, add Justin Hanney to developers list in About dialog

### PR DESCRIPTION
**Pull request by JoePfeiffer**
Test jar at https://drive.google.com/file/d/1k5IGTelyPsUqeVsYCRTRWrrg1DRVQ8Y7/view?usp=sharing

### CODE REVIEW

**Comment by hcraigmiller** [comment]
Functions as expected, no anomalies found.
[Windows 11 Pro, Version 21H2, Build 22000.434, Windows Feature Experience Pack 1000.22000.434.0]

File: core/resources/build.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 # The OpenRocket build version
-build.version=20.11.alpha.16
+build.version=22.xx.BETA
```
Comment (itshanney) [inline core/resources/build.properties:3]: Might as well commit to 22.0 as the release number. The .xx is too ambiguous, so hammer it home :)

Comment (JoePfeiffer) [reply core/resources/build.properties:3]: The plan is to replace the xx with the month number when this release is final.

Comment (SiboVG) [reply core/resources/build.properties:3]: Also: don't use capitals for beta, but make it e.g. 22.02.beta.01, so it's in sync with how the software update checker works

**Comment by JoePfeiffer** [comment]
Wait a minute -- will the updater recognize version 22.04 is later than 22.xx.beta.01? Letters come after numbers in the sort order.

**Comment by SiboVG** [comment]
> Wait a minute -- will the updater recognize version 22.04 is later than 22.xx.beta.01? Letters come after numbers in the sort order.

If xx is smaller than than or equal to 04, then yes, 22.04 should be recognized as being later 22.xx.beta.01

**Comment by JoePfeiffer** [comment]
I meant literally the string "xx", as it currently appears in the version string.

**Comment by SiboVG** [comment]
> I meant literally the string "xx", as it currently appears in the version string.

Then it would simply give a warning dialog that it cannot recognized the version tag, because 'xx' is an unrecognized string.

**Pull request merged**